### ISSUE TITLE
Creating list-items in list-items

### ISSUE THREAD

**Issue by jjmurre** [fraywing/textAngular#461]
I want to create sublists. I am doing the following actions:
- create an ordered list
- select some of the items
- hit the 'indent' button

I want the following to happen:
```
<ol>
<li>a</li>
<li>b</li>
<li>c</li>
</ol>
```
Should become:
```
<ol>
<li>
   a
    <ol><li>b</li></ol>
</li>
<li>c</li>
</ol>
```
However, I get:
```
<ol>
<li>a</li>
    <ol><li>b</li></ol>
<li>c</li>
</ol>
```
That is not a sublist, but a new ordered list in the middle of the initial list.
Is there a way in textAngular to create sublists?

**Comment by episodeyang** [comment]
Hi, this is a browser bug that happens consistently in contenteditable.

I made a work around in this pull-request, but I haven't had time to write the test. If you need this in a hurry you can consider copying the code.

I might come back to fix this PR at somepoint, since list support is very important, but it might take a little while. #442

**Comment by SimeonC** [comment]
This will be fixed in #442 and v1.3.1
